src/components/teasers/teaser.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Teaser } from './teaser';
+import { Astronaut } from '../icons/illustrations/astronaut';
+import { SpaceShuttle } from '../icons/illustrations/space-shuttle';
+
+const icons = { Astronaut: <Astronaut />, SpaceShuttle: <SpaceShuttle /> };
+
+export default {
+  component: Teaser,
+  title: 'Components/Teaser',
+  parameters: {},
+  argTypes: {
+    icon: {
+      options: Object.keys(icons),
+      mapping: icons,
+    },
```

Comment:
oh boy, I didn't know about that. I searched for a mapping like these and instead I did some weird manual mapping in my latest PR. You should have warned me and catch this on my PR 😅 I will go ahead and update it now I know how to map values properly. Thank you!

https://github.com/satellytes/satellytes.com/blob/4b842f651a6ac62c31558a2eb90f93c2802fcd3a/src/components/callout/callout.stories.tsx#L34-L50